old-tutorials-in-development/tutorials-in-development/R/dc-spatio-temporal-intro/04-intro-CRS-projection.Rmd

```diff
@@ -463,17 +463,18 @@ the graphic below optimize?
 ## north...
 
 # What each CRS optimizes:
-## Mercator:
-## ALbers Equal Area
-## UTM Zone 11n
-## Geographic WGS84 (lat/lon):
+## Mercator: distance
+## Albers Equal Area: area
+## UTM Zone 11n: distance
+## Geographic WGS84 (lat/lon): shape
 ```
 
 ***
 
 <figure>
-    <a href="https://source.opennews.org/media/cache/b9/4f/b94f663c79024f0048ae7b4f88060cb5.jpg">
-    <img src="https://source.opennews.org/media/cache/b9/4f/b94f663c79024f0048ae7b4f88060cb5.jpg">
+    <a href="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/geospatial-skills/USMapDifferentProjections_MCorey_Opennews-org.jpg">
+    <img src="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/geospatial-skills/USMapDifferentProjections_MCorey_Opennews-org.jpg"
+    alt="Four maps of the United States each distorted differently.">
```

Comment:
Could you please make this alt text a bit more detailed - explaining that these are four different projections, which are each good for their own purposes, but lead to different types of distortions